contents/blog/why-redesign.mdx

```diff
@@ -0,0 +1,171 @@
+---
+title: Why are we redesigning PostHog?
```

Comment:
The "Why are we" form has some doubt in it. I think we could drop the question mark to exude more confidence
```suggestion
title: Why we're redesigning PostHog
```

Reply:
Agree with above.